navpy/core/navpy.py

```diff
@@ -920,7 +920,7 @@ def ecef2lla(ecef, latlon_unit='deg'):
         Rew,Rns = earthrad(lat,lat_unit='rad')
         h = p/np.cos(lat) - Rew
         
-        err = np.arctan2(z*(1+wgs84._ecc_sqrd*Rew*np.sin(lat)/z),p) - lat
+        err = np.arctan2(z+wgs84._ecc_sqrd*Rew*np.sin(lat),p) - lat
```

Comment:
Any thoughts @adhika as to why the original equation was factored as such?  Might there have been some convergence/numerical reason for that?  Do we need to update the reference?

Reply:
I don't recall. Were we just trying to be fancy? 

I do not have access to Jekeli's book anymore so I am not sure how it's written in that book but I doubt it would overlook this division by zero case. The iteration is also vaguely described in Groves section 2.3.2 Curvilinear Position. I will try to derive them tomorrow again if I still remember how to do math.

Reply:
I don't recall either but did find some hints.

I checked my references and found a lead in the `gnss_ins_functions` from GNSS Applications and Methods.  There is an implementation and reference for the ECEF to LLA conversion. Two things that stood out as I tested it in Octave:
1. The implementation is not iterative.
2. Calling the method for the equator example leads to a *division by zero* warning
Octave handles the division by zero gracefully since it replaces the intermediate value with `inf`.  That in turn allows the arc-tangent to compute the proper value.

I suspect our implementation in navpy has the same assumption (e.g. came from MATLAB/Octave).  If we could identify the divide by zero case and replace the result with `inf` then the answer would be correct.

I looked at numpy and found a possible solution to the issue using `np.divide()`.  Hence, we replace the original line with:
```suggestion
err = np.arctan2(np.divide(z*(1+wgs84._ecc_sqrd*Rew*np.sin(lat), z)),p) - lat
```

Reply:
I digged into my old digital files and found this from Jekeli. I suppose we were not that crazy!

![image](https://user-images.githubusercontent.com/5687421/83970892-0e2e6c80-a88d-11ea-941f-abbc8831b633.png)

Reply:
Nice find!